chalice/deploy/packager.py

```diff
@@ -27,6 +28,9 @@
 OptStr = Optional[str]
 OptBytes = Optional[bytes]
 
+LOGGER_NAME = 'packager'
+logger = logging.getLogger(LOGGER_NAME)
```

Comment:
You can just use `__name__` here instead of `LOGGER_NAME`. This allows the file to get moved around without having to update the string.

Reply:
Yeah I'm fine with just using `__name__` and having this be part of our standard debug logs.